README.md

```diff
@@ -14,15 +14,16 @@ API Docs: https://turnkey.readme.io/
-| [`with-offline`](/examples/with-offline/)                              | Sign a Turnkey request in offline context                                                                          |
-| [`demo-consumer-wallet`](https://github.com/tkhq/demo-consumer-wallet) | A minimal consumer wallet app powered by Turnkey and WalletConnect                                                 |
+| Example                                                                     | Description                                                                                                        |
+| --------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| [`with-ethers`](/examples/with-ethers/)                                     | Create a new Ethereum address, then sign and broadcast a transaction using the Ethers signer with Infura           |
+| [`with-cosmjs`](/examples/with-cosmjs/)                                     | Create a new Cosmos address, then sign and broadcast a transaction on Celestia testnet using the CosmJS signer     |
+| [`with-solana`](/examples/with-solana/)                                     | Create a new Solana address, then sign and broadcast a transaction on Solana's devnet                              |
+| [`with-gnosis`](/examples/with-gnosis/)                                     | Create new Ethereum addresses, configure a 3/3 Gnosis safe, and create + execute a transaction from it             |
+| [`with-uniswap`](/examples/with-uniswap/)                                   | Sign and broadcast a Uniswap v3 trade using the Ethers signer with Infura                                          |
+| [`with-nonce-manager`](/examples/with-nonce-manager/)                       | Create a new Ethereum address, then sign and broadcast multiple transactions in a sequential or optimistic manner. |
+| [`sweeper`](/examples/sweeper/)                                             | Sweep funds from one address to a different address                                                                |
+| [`deployer`](/examples/deployer/)                                           | Compile and deploy a smart contract                                                                                |
+| [`with-offline`](/examples/with-offline/)                                   | Sign a Turnkey request in offline context                                                                          |
+| [`demo-consumer-wallet`](https://github.com/tkhq/demo-consumer-wallet)      | A minimal consumer wallet app powered by Turnkey and WalletConnect                                                 |
+| [`with-federated-authentication`](/examples/with-federated-authentication/) | A nextjs app that demonstrates how to use Turnkey to build a federated, webauthn powered authentication flow       |
```

Comment:
Nit: let's rename this to `with-federated-passkeys` or `with-end-user-passkeys`? It'll be more concrete. When I read "federated authentication" I don't really have a good sense of what this example is about.